internal/task/task.go

```diff
@@ -0,0 +1,62 @@
+package task
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/flynntdev/go_final_project/api"
+	"github.com/flynntdev/go_final_project/config"
+)
+
+type Task struct {
+	ID      string `json:"id"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+func (t *Task) CheckID() error {
+	if t.ID == "" {
+		return fmt.Errorf(`{"error":"Не указан индификатор задачи"}`)
```

Comment:
Опять же это не слой апи, тут свои внутренние ошибки
```suggestion
		return fmt.Errorf("Не указан индификатор задачи")
```